.gitignore

```diff
@@ -8,10 +8,12 @@
 *.swo
 *.swn
 packetbeat.test
- *.dev.yml
+*.dev.yml
+*.cov
 
 /coverage/
 /docs/html_docs
+cover/
```

Comment:
cover should not be in here, as coverage already is. The folders and commands were renamed.